src/main/java/dev/j3fftw/litexpansion/utils/LoreBuilderDynamic.java

```diff
@@ -0,0 +1,73 @@
+package dev.j3fftw.litexpansion.utils;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.MachineTier;
+import io.github.thebusybiscuit.slimefun4.core.attributes.MachineType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactivity;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * This utility class provides a few handy methods and constants to build the lore of any
+ * {@link SlimefunItemStack}. It is mostly used directly inside the class {@link SlimefunItems}.
+ *
+ * @author TheBusyBiscuit
+ *
+ * @see SlimefunItems
+ *
+ */
+public final class LoreBuilderDynamic {
+
+    private static double t = SlimefunPlugin.getCfg().getInt("URID.custom-ticker-delay");
```

Comment:
This should be an int

Reply:
t takes the place of the 12 I used as an example on discord, I have it as a double to prevent the divide by zero problem

Reply:
But the actual type of this is int. Just cast to double later on